docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+- Tizen Porting Layer for EGL (TPL-EGL)
+  - libwayland-egl-tizen
+    - Added wl_egl_window_tizen_set_window_serial function.
+  - libtpl-egl
+    - Added tpl_surface_create_with_num_buffers function.
+    - Added tpl_display_get_with_backend_type function.
+- Vulkan
+  - Added support for VK_KHR_incremental_present.
+  - Added support for using OldSwapchain.
+
+### Fixes
+
+- Enlightenment (Display Server)
+  - Many code defects detected by the static analysis tool have been fixed.
+  - Fixed wrong input region of the window after the screen is rotated to the 90 degrees or 270 degrees.
+  - Fixed unwanted focus set to a window that is specified as 'focus_skip_set' when it is mapped.
+
+
+## Graphics Engine
+
+### New and changed features
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - Support for notification to inform Actor’s children order change has been added.
+    - Support for changing rendering behavior has been added.
+    - A mechanism to specify a callback on every frame has been added.
+    - A CULLED property has been added to Actor.
+  - Application
+    - IdleEnterer callback has been added.
+  - Input
+    - An API has been added to identify right and left mouse button click.
+    - Support for enabling the text prediction has been added.
+    - Support for keyboard repeat information has been added.
+  - Text
+    - Support for software styling has been added.
+  - Timer
+    - API set to pause and resume the timer has been added.
+  - Control and Visual
+    - WebView control has been added.
+    - Support for FlexLayout, AbsoluteLayout and GridLayout has been added.
+  - 3D Model
+    - Support for GL Transmission Format (glTF) has been added.
+  - Performance
+    - Changed not to render transparent texts.
+  - NUI (C# Interface)
+    - Added support for WebView.
+- Simple DirectMedia Layer (SDL)
+  - Upgrade to 2.0.8.
+  - Added SDL sub-module as SDL-ttf.
+  - Added new features.
+    - Auxiliary Hint.
+    - Focus Skip.
+
+### Fixes
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - RenderTask bug to waiting forever has been fixed.
+    - Depth buffer clear bug has been fixed.
+    - BlendFunc has been fixed in case of a non-premultiplied format.
+  - Window
+    - Flickering issue has been fixed when a window is resized.
+  - Control
+    - Some bugs about layout have been fixed.
+  - NUI (C# Interface)
+    - Minor bugs (For example wrong position, not shown) of Layout have been fixed.
+    - RaiseToTop in View class bug has been fixed.
+    - XamlResource memory leak error has been fixed.
+    - VisualFittingMode bug in VisualMaps has been fixed.
+- Evas Rendering Engine.
+  - Bug of rotation is fixed in Evas GL Engine.
+  - Added thread safety patch to Evas TBM Engine.
+  - Bug of image downscaling is fixed in Evas GL Engine.
+  - Added the patch is TBM Queue is used per TBM surface in Evas TBM Engine.
+-  Simple DirectMedia Layer (SDL)
+  - Bug of pause/resume in multi-windows is fixed.

```

Comment:
Can we rephrase as:
Bug in pause or resume in multi-windows is fixed.

Reply:
will accept